manifests/0000_90_kube-controller-manager-operator_03_servicemonitor.yaml

```diff
@@ -42,3 +42,21 @@ spec:
       for: 15m
       labels:
         severity: critical
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: kube-controller-manager-operator
+  namespace: openshift-kube-controller-manager-operator
+spec:
+  groups:
+  - name: cluster-version
+    rules:
+    - alert: KubeControllerManagerDown
+      annotations:
+        message: KubeControllerManager has disappeared from Prometheus target discovery.
+      expr: |
+        kube_poddisruptionbudget_status_expected_pods <= on(poddisruptionbudget, service) kube_poddisruptionbudget_status_desired_healthy
+      for: 15m
```

Comment:
This seems pretty reasonable, how does this compare to others.

Reply:
It's similar to the others, even though the bug mentioned 5 min, I went with what the other alerts usually have.

Reply:
That's fine for me.